sdk/storage/azure-storage-queue/azure/storage/queue/_queue_client.py

```diff
@@ -708,7 +710,7 @@ def update_message(
 
         :param message:
             The message object or id identifying the message to update.
-        :type message: str or ~azure.storage.queue.QueueMessage
+        :type message: str or Optional[~azure.storage.queue.QueueMessage]
```

Comment:
Perhaps this was in response to the comment below? I don't think we want to make this `Optional`.

Reply:
Yep that was a false starter for the other comment